src/Land/Model/source.jl

```diff
@@ -1,4 +1,30 @@
 #### Land sources
+export PhaseChange
+
+function heaviside(x::FT) where {FT}
+    if x > FT(0)
```

Comment:
I don't know if it matters in this context, but isn't this function usually defined as x >= 0 == 1 (or sometimes with the half approx 0 == 0.5)?

Reply:
Yeah, I think there's maybe ambiguity at the discontinuity. We use `heavisided(x) = (x > 0) * x` in thermo 🤷🏻‍♂️

Reply:
will change to what Jake suggested! I think that is the convention. Charlie, looks like thermo is using something like a ReLU: https://en.wikipedia.org/wiki/Rectifier_(neural_networks)

Reply:
Hah, yeah, maybe that's a better name